XenModel/Messages.resx

```diff
@@ -4644,9 +4680,24 @@ Would you like to eject these ISOs before continuing?</value>
   <data name="ENABLE_PLAIN" xml:space="preserve">
     <value>Enable</value>
   </data>
+  <data name="ENABLE_PVS_READ_CACHING_BUTTON" xml:space="preserve">
+    <value>E&amp;nable PVS Read-caching</value>
+  </data>
+  <data name="ENABLE_PVS_READ_CACHING_MENU" xml:space="preserve">
+    <value>Ena&amp;ble PVS Read-caching</value>

```

Comment:
I suggest adding "..." at the end of the menu text for this one, as it is opening a dialog.